### DISCUSSION TITLE
Save scroll position

### DISCUSSION THREAD

**Discussion by MarcoMruz** [dailydotdev/daily#728] [Feature Requests]
Would be nice to save an end user scrolled position after closing the article TLDR. Because when end user have scrolled through 20 articles in list it can be cumbersome to go through the whole list again till he gets to the exact same place where he was before he clicked on an article

**Comment by idoshamun** [comment]
Hi @MarcoMruz,

Thanks for the suggestion. It should actually be implemented, so we probably have an issue with it.
Can you please share more information about your device?

**Reply by MarcoMruz** [reply]
Hi, @idoshamun 

I am using iPhone SE 2020 version - iOS 16.0
Using the PWA version

**Reply by idoshamun** [reply]
Thanks a lot! Forwarded to the team :)

**Comment by randy** [comment]
Same issue with the PWA on iPhone 13 Pro and iOS 16. Came specifically to request this as I find it infuriating.

**Comment by idoshamun** [comment]
@randy @MarcoMruz the issue is now fixed! 🎉

**Reply by randy** [reply]
Yes, yes it is! Thank you!
